pde_superresolution/WENO.py

```diff
@@ -0,0 +1,71 @@
+import numpy as np
```

Comment:
minor style notes: can you please add the Apache copyright header, add `__future__` imports and consistently use spaces instead of tabs in this file?